newIDE/app/src/InstancesEditor/InstancePropertiesEditor/index.js

```diff
@@ -31,8 +33,10 @@ export default class InstancePropertiesEditor extends React.Component<Props> {
       valueType: 'string',
       disabled: true,
       getValue: (instance: gdInitialInstance) => instance.getObjectName(),
-      setValue: (instance: gdInitialInstance, newValue: string) =>
-        instance.setObjectName(newValue),
+      setValue: (instance: gdInitialInstance, newValue: string) => {
+        this.props.unsavedChanges.triggerUnsavedChanges();
```

Comment:
I'm thinking that these functions should not have to deal with updating the unsaved changes at all. Their job is to update a value, not to trigger something related to an editor... they might not even be used in an editor in case they are extracted from this.

It's also quite verbose to have to put triggerUnsavedChanges everywhere. Can we instead put this in PropertiesEditor (or something like this), at the place where setValue is called?

Reply:
(See `_onInstancesModified` in `PropertiesEditor`, might be a good candidate.)